### PR TITLE
feature: add support to PUT meta data

### PR DESCRIPTION
**Pull request by sbender9**

### CODE REVIEW

File: src/put.js [hunk 1]
```diff
@@ -59,6 +61,34 @@ module.exports = {
...
+      try {
+        defaults = skConfig.readDefaultsFile(app)
+      } catch (e) {
+        defaults = {
+          vessels: {
+            self: {}
+          }
+        }
+      }
+      const full = context + '.' + path
```
Comment (tkurki) [inline src/put.js]: `full` refers to _the full model_ in my head, maybe `pathWithContext`?

File: src/put.js [hunk 2]
```diff
@@ -59,6 +61,34 @@ module.exports = {
...
+      _.set(defaults, full, value)
+      _.set(app.config.defaults, full, value)
```
Comment (tkurki) [inline src/put.js]: If defaults data is in memory and we need to change that why do we read the defaults from the file? Or in reverse: why are we modifying defaults data read from the file and `app.config.defaults` separately and not just setting `app.config.defaults = defaults`?

Comment (sbender9) [reply src/put.js]: good point. will fix...

File: src/put.js (hunk 1 above)
Comment (sbender9) [reply src/put.js]: done

File: src/put.js (hunk 2 above)
Comment (sbender9) [reply src/put.js]: Cleaned that stuff up, please take a look.

**Comment by sbender9** [comment]
I updated to allow to PUT to */meta

**Pull request merged**